### ISSUE TITLE
LinearPredictor.Statistics is always null

### ISSUE THREAD

**Issue by klausmh** [dotnet/machinelearning#1006]
### System information

- **OS version/distro**: Win 10 / ML.NET 0.5.0
- **.NET Version (eg., dotnet --info)**: 4.6.2

### Issue

- **What did you do?**

I train a model using logistic regression:
```
var trainer = new LogisticRegression(
                        environment,
                        new LogisticRegression.Arguments() { ShowTrainingStats = true });
var trainRoles = ...
var predictor.Train(trainRoles)
```

- **What happened?**
`predictor.Statistics` is null

- **What did you expect?**
To be able to retrieve model statistics

### Source code / logs

`LogisticRegression.CreatePredictor` does not pass its `_stats` `LinearModelingStatistics` field when it creates a `LinearBinaryPredictor`: https://github.com/dotnet/machinelearning/blob/1e7b8be855210f2bd8adbd532396a1840a20541d/src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LogisticRegression.cs#L327.

If I do change that, then `predictor.Statistics.GetCoefficientStatistics` still returns null. It appears that you calculate the inverted Hessian, but then do not use it to calculate standard errors (and pass these in the constructor of `LinearModelStatistics'): https://github.com/dotnet/machinelearning/blob/1e7b8be855210f2bd8adbd532396a1840a20541d/src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LogisticRegression.cs#L189.

**Comment by montebhoover** [comment]
So there are two separate issues here:

1. _stats is not passed to predictor: captured in issue #1205 
2. GetCoefficientStatistics are no longer calculated since decoupling from MKL: captured in issue #612

**Comment by GalOshri** [comment]
@klausmh, would you be able to take a look at the discussion in #612 and the relevant PR (#1392) to make sure the solution will fulfill your requirements?

**Comment by sfilipi** [comment]
Closed with PR #1392
